src/components/starRating.tsx

```diff
@@ -1,56 +1,58 @@
 // ----- Imports ----- //
 
-import React, { ReactNode, ReactElement } from 'react';
+import React, { ReactNode, FC } from 'react';
 import { css } from '@emotion/core';
-import { brandAltBackground, text } from '@guardian/src-foundations/palette';
-import { remSpace } from '@guardian/src-foundations';
-
+import { brandAltBackground, brandAltLine } from '@guardian/src-foundations/palette';
+import { SvgStar } from '@guardian/src-icons';
 import { Item } from 'item';
-import { icons, darkModeCss } from 'styles';
+import { darkModeCss } from 'styles';
 import { Design } from '@guardian/types/Format';
 
 
 // ----- Subcomponents ----- //
 
 const starStyles = css`
-    ${icons}
     background-color: ${brandAltBackground.primary};
-    font-size: ${remSpace[5]};
-    line-height: 1;
     display: inline-block;
-    padding: 0 0.2rem ${remSpace[1]};
-    color: ${text.primary};
-
-    &:nth-of-type(1) {
-        padding-left: ${remSpace[1]};
-    }
+    line-height: 0;
 
     &:nth-of-type(5) {
-        padding-right: ${remSpace[1]};
+        padding-right: 2px;
+    }
+;
+    svg {
+        margin: 0 -0.125rem;
+        height: 2rem;
```

Comment:
This brings the size a bit closer to dotcom and the current apps-rendering articles:

```suggestion
        height: 1.75rem;
```